boost.sh

```diff
@@ -948,24 +966,34 @@ buildBoost_Linux()
     mkdir -p $OUTPUT_DIR
     echo > ${OUTPUT_DIR}/linux-build.log
 
-    for BITS in 64 32; do
+    for BITS in 32 64; do
         for VARIANT in debug release; do
             echo Building $VARIANT $BITS-bit Boost for Linux
 
             if [[ $BITS == 64 ]]; then
                 LIBDIR_SUFFIX=x86_64
+                ARCH=x86
+                TOOLSET=gcc-6.3.0~x86_64
+                ABI=sysv
             else
-                LIBDIR_SUFFIX=x86
+                LIBDIR_SUFFIX=arm_gnueabihf
```

Comment:
Why 32 bit builds are only for arm?

Reply:
i686 seems to be widely unsupported on linux these days

Reply:
We still build it and i dont want to break the release flow for no reason.